### DISCUSSION TITLE
Adding Contributions graph/chart thingie to the profile section on the mobile app.

### DISCUSSION THREAD

**Discussion by abhisheksahdev** [community/community#11337] [Mobile]
1. Add Contributions graph.. to the profile section of the app.

2. A widget for the Android home screen like this one of the [play store](https://play.google.com/store/apps/details?id=jp.co.chlorocube.githubcontributions)

**Comment by vdanbv** [comment]
setup

**Comment by unique-EJ** [comment]
Contributions: [3959](https://github.com/github/feedback/discussions/3959)

**Comment by herculeskan** [comment]
Hi! Thanks for developing this app, the next year I'm going to focus more in being more contribute to GitHub
